bfdd/bfd_packet.c

```diff
@@ -697,11 +697,26 @@ int bfd_recv_cb(struct thread *t)
 
 		/* Handle poll finalization. */
 		bs_final_handler(bfd);
-	} else {
-		/* Received a packet, lets update the receive timer. */
-		bfd_recvtimer_update(bfd);
```

Comment:
will there be a performance hit from checking this for every received packet? 3x/second, or whatever the timers are set to?

Reply:
`bfd_recvtimer_update` is already called for every packet – either from this function, or from `bs_final_header`. The call is just moved to another place.

The only change is that the following code is now called for every packet:
```
	if (bfd->timers.required_min_rx > bfd->remote_timers.desired_min_tx)
 		bfd->detect_TO = bfd->remote_detect_mult
 				 * bfd->timers.required_min_rx;
 	else
 		bfd->detect_TO = bfd->remote_detect_mult
 				 * bfd->remote_timers.desired_min_tx;
```
It's just a couple of simple operations, so won't add much processing time. Anyway, such behavior is required in the RFC, so we can't do anything different here.

Reply:
thanks